Eastern-Research-Group/expert-query
# Feature/61 db connection

## Related Issues:
* [EQ-61](https://jira.epa.gov/browse/EQ-61)

## Main Changes:
* Wired up the database connection on the app side.
  * This will use the latest active schema.
  * We have two main API paths `/api` will be used for non-database type services and `/data` will be used for all database type service calls.

## Steps To Test:
1. Create a `.env.local` based the `.env.example` file, the data values should be similar to the ones on the `etl` side.
2. Run the app
3. Verfiy the app loads and works
4. Navigate to http://localhost:9090/data/assessments
5. Verify json data is displayed
6. In the database, change the active schema
7. Reload the url in step 4
8. Verify the json data is different from the data in step 5.
